cmd/snap-confine/selinux-support.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef SNAP_CONFINE_SELINUX_SUPPORT_H
+#define SNAP_CONFINE_SELINUX_SUPPORT_H
+
+/**
+ * Set security context for the snap
```

Comment:
Could you please extend this comment and say e.g. "Set security context for the snap to support transition from snappy_confine_t to unconfined_service_t"?

Reply:
I'm pretty sure the goal is to eventually make this do more than that in the future...

Reply:
Here that is fine I think.